packages/hash/frontend/src/pages/[account-slug]/index.page.tsx

```diff
@@ -46,21 +48,45 @@ const Page: NextPageWithLayout = () => {
     );
   }
 
-  const workspaceName = ownWorkspace ? "your" : `${thisOrg!.name}'s`;
-
   return (
     <>
-      <Box component="header" mt={1.5}>
-        <Typography mb={2} variant="h1">
-          Hi, {user.properties.preferredName}!
+      <Typography mb={3} variant="h2">
+        Welcome to HASH
+      </Typography>
+      <Box sx={{ maxWidth: "75ch" }}>
+        <Typography mb={3}>
+          HASH is an open-source, data-centric, all-in-one workspace built atop
+          the open <Link href="https://blockprotocol.org">Block Protocol</Link>,
+          allowing users to easily add new block types and functionality to
+          their workspaces.
```

Comment:
```suggestion
          the open <Link href="https://blockprotocol.org">Block Protocol</Link>.
```

Reply:
Updated in [c40fbef](https://github.com/hashintel/hash/pull/610/commits/c40fbef90c42ce84c851ca57ec9676a172480d84)